packages/ckeditor5-list/src/listconfig.ts

```diff
@@ -139,4 +139,30 @@ export interface ListPropertiesStyleConfig {
 	 * @default false
 	 */
 	useAttribute?: boolean;
+
+	/**
+	 * Turn off style feature for the given list type.
+	 *
+	 * ```ts
+	 * {
+	 * 	list: {
+	 * 		properties: {
+	 * 			styles: {
+	 * 				exclude: [ 'ul' ]
+	 * 			},
+	 *
+	 * 			// ...
+	 * 		}
+	 * 	},
+	 *
+	 * 	// ...
+	 * }
+	 * ```
+	 *
+	 * **Note**: This configuration works only with
+	 * {@link module:list/listproperties~ListProperties list properties}.
+	 *
+	 * @default []
+	 */
+	exclude?: Array<'ul' | 'ol'>;
```

Comment:
1. Why exclude? Is opt-out really preferred over opt-in?
2. I know using 'ul' and 'ol' was suggested in ticket comment, but those lists are named 'bulleted' and 'numbered'.
3. How above points play with multi-level lists? Does both make sense if this plugin is also present?

Reply:
1. I think opt-out would be more preferable, because of keeping backward compatibility, if users want to have styles for both types of list they need to update their configs.
2. Sure, I think it's a good idea with using `bulleted` and `numbered` list 🙂.
3. Multi level list it's a separate feature and AFAIK multi level list doesn't have any styles at the moment, then I didn't consider this type of list.